x-pack/plugins/alerting/public/pages/maintenance_windows/translations.ts

```diff
@@ -568,3 +568,24 @@ export const EXPERIMENTAL_DESCRIPTION = i18n.translate(
 export const UPCOMING = i18n.translate('xpack.alerting.maintenanceWindows.upcoming', {
   defaultMessage: 'Upcoming',
 });
+
+export const UPGRADE_TO_PLATINUM = i18n.translate(
+  'xpack.alerting.maintenanceWindows.callout.updateToPlatinumTitle',
+  {
+    defaultMessage: 'Upgrade to an appropriate license',
```

Comment:
This title seems a little "friendlier":
```suggestion
    defaultMessage: 'To access maintenance windows, you must upgrade your license',
```